src/client/digital.identity.client.js

```diff
@@ -18,6 +18,15 @@ class DigitalIdentityClient {
     this.digitalIdentityService = new DigitalIdentityService(sdkId, pem, options);
   }
 
+  /**
+   *
+   * @param {string} receiptId
+   * @returns {Promise<Receipt>}
+   */
+  async getReceipt(receiptId) {
```

Comment:
It seems inconsistent to be using `get` here while the other client methods use `fetch`. I know that we make a distinction between `getReceipt` and `fetchReceipt` at the service level but that doesn't seem like a clear way of differentiating those two methods. Maybe they should be called `fetchReceipt` and `fetchEncryptedReceipt` in `DigitalIdentityService`?

Reply:
I agree on the `get < fetch` side of things. `getReceipt` should probably become `fetchReceipt`.

Regarding the other methods that I introduced here. I'm not even sure if we want to expose those on the client. If they are only used by this `getReceipt`. Do we think it's useful for integrators?

Reply:
Oh, I wasn't suggesting we expose the other methods on the client. I was just suggesting we rename the methods in `DigitalIdentityService` to be clearer and more consistent with this renaming in the client. So maybe,
* `DigitalIdentityService.getReceipt` -> `DigitalIdentityService.fetchAndDecryptReceipt` and
* `DigitalIdentityService.fetchReceiptById` -> `DigitalIdentityService.fetchEncryptedReceipt`

Reply:
Ah ok.

Yes that sounds like a good idea. Then the exposed method via the client would be just `fetchReceipt`, since the details are not super important to an integrator.